src/language.c

```diff
@@ -0,0 +1,99 @@
+#include "language.h"
+#include <string.h>
+
+typedef struct langPair_s
+{
+  const char *str;
+  RSLanguage lang;
+} langPair_t;
+
+langPair_t __langPairs[] = {
+  { "arabic",     RS_LANG_ARABIC },
+  { "armenian",   RS_LANG_ARMENIAN },
+  { "basque",     RS_LANG_BASQUE },
+  { "catalan",    RS_LANG_CATALAN },
+  { "danish",     RS_LANG_DANISH },
+  { "dutch",      RS_LANG_DUTCH },
+  { "english",    RS_LANG_ENGLISH },
+  { "finnish",    RS_LANG_FINNISH },
+  { "french",     RS_LANG_FRENCH },
+  { "german",     RS_LANG_GERMAN },
+  { "greek",      RS_LANG_GREEK },
+  { "hindi",      RS_LANG_HINDI },
+  { "hungarian",  RS_LANG_HUNGARIAN },
+  { "indonesian", RS_LANG_INDONESIAN },
+  { "irish",      RS_LANG_IRISH },
+  { "italian",    RS_LANG_ITALIAN },
+  { "lithuanian", RS_LANG_LITHUANIAN },
+  { "nepali",     RS_LANG_NEPALI },
+  { "norwegian",  RS_LANG_NORWEGIAN },
+  { "portuguese", RS_LANG_PORTUGUESE },
+  { "romanian",   RS_LANG_ROMANIAN },
+  { "russian",    RS_LANG_RUSSIAN },
+  { "serbian",    RS_LANG_SERBIAN },
+  { "spanish",    RS_LANG_SPANISH },
+  { "swedish",    RS_LANG_SWEDISH },
+  { "tamil",      RS_LANG_TAMIL },
+  { "turkish",    RS_LANG_TURKISH },
+  { "yiddish",    RS_LANG_YIDDISH },
+  { "chinese",    RS_LANG_CHINESE },
+  { NULL,         RS_LANG_UNSUPPORTED }
+};
+
+const char *RSLanguage_ToString(RSLanguage language) {
```

Comment:
Can we use `language` as the index in `__langPairs` (checking it is not beyond `RS_LANG_UNSUPPORTED`)
And returning the `str` in this slot?